modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageItem.es.js

```diff
@@ -105,10 +124,34 @@ function Item({totalCount, ...instance}) {
 		setSelectedItems(updatedItems);
 	};
 
-	const [slaResult] = slaResults;
+	const criticalSlaResults = slaResults.slice(0, 2).map((criticalSla) => {
+		const datetimeOverdueFormatted = getDueDateFormatted(
```

Comment:
Hey @jotascript, Hey @jotascript, can you rename this to slaResults or something different?

They are just the first two SLAResults and their due date can happen in 2 years for example. So, depending on the context, they cannot be considered critical. It makes sense?

//cc @naraandrade